docs/cli/ilab-profile-set.md

```diff
@@ -0,0 +1,55 @@
+# `ilab profile set`
+
+### Usecase
+
+1. User needs to set:
+    a. Model Path for SDG, Eval, Serving, Training
+    b. GPUs for SDG, Eval, Serving, Training
+    c. Training Config Per-GPU (based on vram)
+2. User wants us to detect all of the above for them and provide a sane config
+
+### Workflow
+
+The user will run `ilab profile set`
```

Comment:
Will this be required before running other `ilab` commands? If not, what is the path for a user that does not do this?